app-emulation/genymotion-bin/genymotion-bin-3.0.4.ebuild

```diff
@@ -0,0 +1,123 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit bash-completion-r1 desktop pax-utils
+
+MY_PN="${PN/-bin}"
+MY_P="${MY_PN}-${PV}"
+BIN_ARCHIVE="${MY_P}-linux_x64.bin"
+
+DESCRIPTION="Complete set of tools that provide a virtual environment for Android"
+HOMEPAGE="https://genymotion.com"
+SRC_URI="${BIN_ARCHIVE}"
+
+LICENSE="genymotion"
+SLOT="0"
+KEYWORDS="-* ~amd64"
+
+DEPEND=""
+RDEPEND="|| ( >=app-emulation/virtualbox-5.0.28 >=app-emulation/virtualbox-bin-5.0.28 )
+	virtual/opengl
+	|| (
+		dev-libs/openssl-compat:1.0.0
+		=dev-libs/openssl-1.0*:0
+	)
+	dev-libs/hiredis:0/0.14
+	sys-apps/util-linux
+"
+BDEPEND=">=dev-util/patchelf-0.9_p20180129"
+
+RESTRICT="bindist fetch"
+S="${WORKDIR}"
+
+QA_PREBUILT="
+	opt/${MY_PN}/*.so*
+	opt/${MY_PN}/imageformats/*.so
+	opt/${MY_PN}/plugins/*.so*
+	opt/${MY_PN}/${MY_PN}
+	opt/${MY_PN}/genyshell
+	opt/${MY_PN}/player
+	opt/${MY_PN}/${MY_PN}adbtunneld
+	opt/${MY_PN}/gmtool
+"
+
+pkg_nofetch() {
+	einfo
+	einfo "Please visit"
+	einfo
+	einfo "  https://www.genymotion.com/download/"
+	einfo
+	einfo "and download "
+	einfo
+	einfo "  ${BIN_ARCHIVE}"
+	einfo
+	einfo "which must be placed in DISTDIR directory."
+	einfo
+}
+
+src_unpack() {
+	cp "${DISTDIR}/${BIN_ARCHIVE}" "${WORKDIR}" || die "cp failed"
+}
+
+src_prepare() {
+	default
+
+	chmod +x ${BIN_ARCHIVE} || die "chmod failed"
+	yes | ./${BIN_ARCHIVE} > /dev/null || die "unpack failed"
+
+	# removed windows line for bashcompletion
+	sed -i "/complete -F _gmtool gmtool.exe/d" "${MY_PN}/completion/bash/gmtool.bash" || die "sed failed"
+
+	# patch to support newer hiredis version (0.14)
+	for i in genymotion genyshell gmtool player libcom.so.1.0.0 librendering.so.1.0.0 ; do
+		patchelf --replace-needed libhiredis.so.0.13 libhiredis.so.0.14 "${MY_PN}/${i}" || die "Unable to patch ${i} for hiredis"
+	done
+}
+
+src_install() {
+	insinto /opt/"${MY_PN}"
+	exeinto /opt/"${MY_PN}"
+
+	# Use qt bundled
```

Comment:
Any chance to use system-Qt?

Reply:
Unfortunately not, if I remove all qt bundled library and I run program I have following error:
`[Genymotion Shell:18945] [fatal] Cannot mix incompatible Qt library (version 0x50906) with this library (version 0x50e01)`
this means the program is compiled with qt 5.9.6